src/app/workflow/workflow.component.ts

```diff
@@ -238,6 +245,24 @@ export class WorkflowComponent extends Entry implements AfterViewInit {
     }
   }
 
+  // Show linked services in the UI
+  private setAvailableTokens(tokens) {
+      const found = tokens.find(token => token.tokenSource === 'zenodo.org');
```

Comment:
`this.zenodoAccountIsLinked = tokens.some(token => token.tokenSource === 'zenodo.org');`

Reply:
done